handbook/policies_and_training.md

```diff
@@ -0,0 +1,349 @@
+    
+    - **Is using a PIP a good solution here?**
+        
+        One possible solution to this is for the People team to introduce a formalized process that managers need to go through in order to terminate an employee. This is often known as a "PIP"—performance improvement plan—and it typically plays out over the course of 30-90 days. Because the process is run by the People team, there is a level of assurance that the manager cannot act exclusively on their own authority.
+        
+        The PIP, though, is deeply flawed. It is extremely ritualized, focused on documentation, and impersonal. In reality, companies tend to use PIPs not to help fix performance issues, but instead to document their reasons for terminating someone *after they have already decided to do so*, purely for legal liability reasons.
+        
+        PIPs have very negative cultural and practical outcomes:
+        
+        - They are fundamentally disingenuous and encourage "double-speak." We value transparency and refuse to create rituals that undermine this value.
+        - They do not actually accomplish their stated goals of employee protection—in practice they serve to protect the company.
+        - They prevent the affected employee from having the time to collect themselves and plan out their next moves. The time we choose to pay in severance, a company that uses PIPs is still asking the employee to spend all of their time on a job from which they will be terminated. We would prefer to move decisively and allow the terminated employee to use this paid time in whatever way was the most helpful **to them**.
+        - The delay in terminating someone reduces the speed with which the organization can solve problems. 90 days of underperformance in a role is a very long time for a company in hypergrowth.
+        
+        For all of these reasons, we don't use PIPs at dbt Labs. We do, however, involve the People team in every single termination that occurs at the company to prevent unilateral decisions from being made by managers.
+        
+    - **What is our preferred way of solving this problem?**
+        
+        When framing this problem in terms of a mismatch between accountability and impact, it becomes more clear what good potential solutions are. The question decomposes down into two distinct sub-questions: 
+        
+        - How do we minimize the impact on the terminated employee?
+        - How do we increase the accountability of managers and the organization as a whole?
+        
+        Minimizing the impact on terminated employees is the primary motivating force behind our severance policy. We have an [aggressive severance policy](https://www.notion.so/Involuntary-Terminations-2ae0d5393458432d87ce270fa7530765) specifically because we want to minimize the impact of a termination on the terminated employee.
```

Comment:
This link is to a trashed doc, suggest removing or linking to the severance section below

Reply:
Testing out to see if my link worked.